GPU/GPUCommon.cpp

```diff
@@ -1808,6 +1808,70 @@ void GPUCommon::Execute_MorphWeight(u32 op, u32 diff) {
 	gstate_c.morphWeights[(op >> 24) - GE_CMD_MORPHWEIGHT0] = getFloat24(op);
 }
 
+void GPUCommon::Execute_ImmVertexAlphaPrim(u32 op, u32 diff) {
+	// Safety check.
+	if (immCount_ >= MAX_IMMBUFFER_SIZE)
+		return;
```

Comment:
Can we log this situation?  Although I'd expect it probably means garbage.

-[Unknown]